lib/application/presentation/schedule_screen.dart

```diff
@@ -0,0 +1,238 @@
+                    child: Transform(
+                      transform: Matrix4.skewX(-0.3),
+                      //dynamic
+                      child: Text(
+                        'THU, OCT 20TH',
+                        style: Theme.of(context).textTheme.titleMedium,
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 10),
+          //buttons for filtration by category
+          SingleChildScrollView(
+            scrollDirection: Axis.horizontal,
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                CategoryButton(
+                  category: ScheduleItemCategory.all,
+                  onPressed: () => {
+                    //implement filtration by category
+                  },
+                ),
+                const SizedBox(width: 16),
+                CategoryButton(
+                  category: ScheduleItemCategory.tech,
+                  onPressed: () => {
+                    //implement filtration by category
+                  },
+                ),
+                const SizedBox(width: 16),
+                CategoryButton(
+                  category: ScheduleItemCategory.ops,
+                  onPressed: () => {
+                    //implement filtration by category
+                  },
+                ),
+                const SizedBox(width: 16),
+                CategoryButton(
+                  category: ScheduleItemCategory.lead,
+                  onPressed: () => {
+                    //implement filtration by category
+                  },
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 10),
+          //list view of schedule items
+          BlocBuilder<ScheduleCubit, ScheduleState>(
+            builder: (context, state) {
+              return Expanded(
+                child: ListView.builder(
+                  itemCount: state.scheduleItems.length,
+                  itemBuilder: (context, index) {
+                    return listBuild(state.scheduleItems[index]);
+                  },
+                ),
+              );
+            },
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget listBuild(ScheduleItem scheduleItem) {
+    //returns a card which we use to set padding between list tiles
+    //each card has a child listTile
+    //listTile's title is a column of three rows, first row is category and time
+    //second row is agenda item title, which has fixed height and set overflow after 2 lines
+    //third row is optional, if there's a hall then it displays the hall
+    //after the title atribute column there's a leading button and an onpressed func
+    return Card(
+      margin: const EdgeInsets.fromLTRB(15, 5, 0, 5),
+      child: ListTile(
+        title: Column(
+          children: [
+            //category and time
```

Comment:
isto nepotrebni komentari, možete ih maknuti, neću više pisati da se ne ponavljam. Generalno komentari su OK tamo gdje bi moglo biti nejasno što kod radi :)